tests/reftests/pin.test

```diff
@@ -1,9 +1,9 @@
 N0REP0
 ### OPAMYES=1
 ### <add-pin.sh>
-nv=$1
-n=${nv%.*}
-v=${nv#*.}
+nv="$1"
+n="${nv%.*}"
+v="${nv#*.}"
```

Comment:
just so you know, double quotes for variable definitions are not necessary.
For example:
```
x=$y/$z
```
is always safe. But:
```
x="$y $z"
```
does require quotes